### PR TITLE
Fixes #18741 - Fix filtering of report messages

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

**Comment by ares** [comment]
actually it seems to be problem in core as well, we could fix it there and keep using core method

**Comment by xprazak2** [comment]
@ares, I know we wanted to fix this in core, but core does not use multiple labels in the table, so it does not break core because of the way it is used and we also filter on different labels - warning, danger, success while core has info, default, warning, danger. But I can try to get this in core if you want me to.

**Comment by ares** [comment]
I think that reusing the code from the core is a small victory, I understand it's not reproducible upstream but I think no one would object adding extra css class to this column and update the JS function to use it. The amount of duplication here seems too high.

File: app/helpers/arf_reports_helper.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ def result_tag(level)
           else
             "warning"
           end
-    "class='label label-#{tag}'".html_safe
+    "class='label label-#{tag} result-filter-tag'".html_safe
```
Comment (ares) [inline app/helpers/arf_reports_helper.rb]: This is all we need from this PR now after https://github.com/theforeman/foreman/pull/4728 was merged. I already tested it works, please remove the rest.

**Comment by xprazak2** [comment]
I kept only the newly added css class, removed the rest.

**Comment by ares** [comment]
Thanks @xprazak2, merging!

**Pull request merged**